src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Security/RequestHeaderConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Security;
+
+public static class RequestHeaderConstants
```

Comment:
Could rename this to `RequestHeaderNames`? This way we can short the fairly verbose property name to simply `PreviewToken`.

Would be accessed like `RequestHeaderNames.PreviewToken`.

Reply:
Good idea, made this change, thanks.